DevTools/HackStudio/Debugger/Debugger.cpp

```diff
@@ -259,4 +267,12 @@ void Debugger::insert_temporary_breakpoint(FlatPtr address)
     m_state.add_temporary_breakpoint(address);
 }
 
+void Debugger::set_requested_debugger_action(DebuggerAction action)
+{
+    pthread_mutex_lock(Debugger::the().continue_mutex());
+    Debugger::the().m_requested_debugger_action = action;
+    pthread_cond_signal(Debugger::the().continue_cond());
+    pthread_mutex_unlock(Debugger::the().continue_mutex());
```

Comment:
Since we are in `Debugger` here already, no need to keep calling `Debugger::the()` :)

Reply:
Also maybe `set_requested_action` is enough for the name, then we don't have to say "debugger" twice when calling it.